### ISSUE TITLE
duplicates of names in configs

### ISSUE THREAD

**Issue by erertertet** [maruohon/tweakeroo#498]
there are two freeCameraPlayerInputs and freeCameraPlayerMovement in Generic and Hotkeys

![image](https://github.com/maruohon/tweakeroo/assets/41342153/7d16a833-5b7f-47fb-b83d-5c24f02a4d8d)

![image](https://github.com/maruohon/tweakeroo/assets/41342153/036f6650-f74b-4a4b-b01b-6c78fee0ace6)

making it impossible to set the description for them separately for the translation in the form of:

`{"config.comment.freecameraplayermovement" : <translation>}`

for both of them separately

**Comment by maruohon** [comment]
Yeah this is an issue in the old code with the split configs that really should be one "hotkeyed boolean" config, and are combined as such in the newer development code.

So for now I would recommend just writing the tooltip in a way that works for both of them, basically maybe mentioning both configs and how the hotkey just toggles the value of the boolean config. I'd rather not go changing anything about them at this point in the old code, as that would then just break any existing translations.

The proper fix is in the new code, whenever that gets into releases in other versions than 1.12.2... That release "breaks everything anyway" (localization, texture pack and partially user config wise) so it's a good point to overhaul and clean up all of the old crud such as stupid config names, stupid hotkey names, multi-use confusing hotkeys, and the lackluster config tooltips etc. that the old code in 1.13+ has currently.

**Comment by erertertet** [comment]
I currently have a temp working txt file for translation which should be able to convert into any form of JSON if the format/naming of configs does not change drastically. But any way I am going to continue work on zh_cn.json

**Comment by maruohon** [comment]
It's not just the names that have changed in the development code. There are many new configs and messages, some of the names have changed, and most of the tooltips have been rewritten and expanded and improved a lot. Although maybe it doesn't affect Tweakeroo nearly as much as it does some of the other mods... although I guess there lots of config tooltips at least in Tweakeroo as well.

I think Tweakeroo is actually the last mod that hasn't received its (full?) name and tooltip updates yet in 1.12.2. And Litematica is probably also still somewhat incomplete. Don't remember about MiniHUD... Item Scroller is supposed to be entirely rewritten so I don't think I did much with it yet either. But malilib changed by far the most, as the old version barely had anything in the lang file, the new code has *a lot* of stuff there now.
